content/cloud-docs/api-docs/configuration-versions.mdx

```diff
@@ -346,6 +352,38 @@ curl \
   https://archivist.terraform.io/v1/object/4c44d964-eba7-4dd5-ad29-1ece7b99e8da
 ```
 
+## Archive a Configuration Version
+
+`POST /configuration-versions/:configuration_version_id/actions/archive`
+
+| Parameter                   | Description                                    |
+| --------------------------- | ---------------------------------------------- |
+| `configuration_version_id`  | The ID of the configuration version to archive. |
+
+This endpoint notifies Terraform Cloud to discard the uploaded `.tar.gz` file associated with this configuration version. This endpoint can only archive configuration versions that were created with the API or CLI, are in an `uploaded` [state](#configuration-version-states), and have no runs in progress. Otherwise, calling this endpoint will result in an error.
+
+Terraform Cloud automatically archives configuration versions created through VCS when associated runs are complete and then re-fetches the files for subsequent runs.
+
+| Status  | Response                  | Reason(s)                                                                                                                                     |
+| ------- | ------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------- |
+| [202][] | none                      | Successfully initiated the archive process.                                                                                                   |
```

Comment:
question: does `[202][]` need to link to a specific url?

e.g.

```suggestion
| [202](https://httpstatuses.com/202) | none                      | Successfully initiated the archive process.                                                                                                   |
```

Reply:
This is actually a [link reference](https://github.com/hashicorp/terraform-website/blob/370f7c7e8dfab06bfaaf6f4babef2faed606313d/content/cloud-docs/api-docs/configuration-versions.mdx?plain=1#L5) to the MDN docs that's defined at [the top of this file](https://github.com/hashicorp/terraform-website/blob/370f7c7e8dfab06bfaaf6f4babef2faed606313d/content/cloud-docs/api-docs/configuration-versions.mdx?plain=1#L5)!

So once the page is rendered, that url gets populated properly ([link to the Vercel rendered preview for proof](https://terraform-website-l1tobs36m-hashicorp.vercel.app/cloud-docs/api-docs/configuration-versions#archive-a-configuration-version)). It definitely looks weird by itself though.